libraries/OTAUpdate/examples/WiFiFirmwareOTA/WiFiFirmwareOTA.ino

```diff
@@ -0,0 +1,127 @@
+/*
+  OTA
+
+  This sketch demonstrates how to make an self OTA Update the UNO R4 WiFi Firmware
+  Upload the Sketch and wait for Serial detach.
+  After the update the new wifi firmware version will be 98.98.98
+
+*/
+
+
+#include "WiFiS3.h"
+#include "OTAUpdate.h"
+#include "root_ca.h"
+#include "arduino_secrets.h"
+
+char ssid[] = SECRET_SSID;    // your network SSID (name)
+char pass[] = SECRET_PASS;    // your network password (use for WPA, or use as key for WEP)
+
+int status = WL_IDLE_STATUS;
+
+OTAUpdate ota;
+static char const OTA_FILE_LOCATION[] = "https://downloads.arduino.cc/ota/UNOR4USBBridge.ino.ota";
+
+/* -------------------------------------------------------------------------- */
+void setup() {
+/* -------------------------------------------------------------------------- */
+  //Initialize serial and wait for port to open:
+  Serial.begin(115200);
+  while (!Serial) {
+    ; // wait for serial port to connect. Needed for native USB port only
+  }
+
+  // check for the WiFi module:
+  if (WiFi.status() == WL_NO_MODULE) {
+    Serial.println("Communication with WiFi module failed!");
+    // don't continue
+    while (true);
+  }
+
+  String fv = WiFi.firmwareVersion();
+  if (fv < WIFI_FIRMWARE_LATEST_VERSION) {
+    Serial.println("Please upgrade the firmware");
+  }
+
+  Serial.print("Current WIFi firmware version: ");
```

Comment:
```suggestion
  Serial.print("Current WiFi firmware version: ");
```
Fix case of letter.

The correct spelling is actually the hyphenated "[Wi-Fi](https://en.wikipedia.org/wiki/Wi-Fi)". This spelling is not used in Arduino product names due to being trademarked, but in my opinion the correct spelling should be used in prose such as this comment where there should be no problem with trademark infringement. But I'll settle for just changing the letter case.